tests/hollow_sphere_gmg.prm

```diff
@@ -0,0 +1,111 @@
+# The 3D HollowSphere benchmark, for which an 
```

Comment:
for all your test/*.prm , add a line at the top, what it tests like "solve the hollow_sphere.prm with GMG"